src/unix/syscall.c

```diff
@@ -1026,7 +955,8 @@ static int try_write_dirent64(tuple root, struct linux_dirent64 *dirp, char *p,
             *read_sofar -= len;
             return -1;
         } else {
-            tuple n = resolve_cstring(root, p);
+            tuple n;
+            resolve_cstring(root, p, &n, 0);
```

Comment:
check for error?

Reply:
No error can occur, since p is a string retrieved from the children of root; and even if there was an error it wouldn't cause any crash because n is only used in a call to u64_from_pointer().